tools/setup_helpers/env.py

```diff
@@ -21,6 +21,8 @@
 
 BUILD_DIR = "build"
 
+LIBTORCH_PKG_NAME = "libtorchsplit"
```

Comment:
I wish we have a better name for this, i.e. libtorch-lite?

Reply:
@PaliC can we give it a name more descriptive of what it's supposed to do? 

E.g. Is this a wrapper around libtorch? Is it the core libtorch library?

Reply:
Yeah this is a placeholder name. I was planning on having a better name in a separate PR after we decide on one. Sadly we gotta look for an open pypi package for it.